odoo/tests/tag_selector.py

```diff
@@ -55,14 +55,16 @@ def check(self, test):
         test_class = test.test_class
         test_tags = test.test_tags | {test_module}  # module as test_tags deprecated, keep for retrocompatibility,
         test_method = test._testMethodName
+        test_module_path = '/'.join(test.__module__.split('.')[2:])
+        test_module_path = f"{test_module_path}.py"
```

Comment:
We can fix at the same time files ending with the same name by adding a leading "/" to the path.
```suggestion
        test_module_path = test.__module__[test.__module__.index('.', 2):].replace('.', '/') + '.py'
```

And on line 34, `file_path = f"/{module}"`.

Reply:
Actually the index start is a character pos so it won't work

But it is True that the leading whitespace is needed to avoid some mismatch! 

We can do `test_module_path = test.__module__.removeprefix('odoo.addons').replace('.', '/') + '.py'`

Reply:
It is ok to merge as-is. `removeprefix` also works.
I wanted pre-pending "/" to both `test_module_path` and `file_path` (above). I think that currently `--test-tags /a.py` matches `odoo.addons.xxx.tests.haha`, because the leading slash is stripped by the regex when parsing the tag.

Reply:
Yes I modified the line

`file_path = f"/{module}"`

Reply:
Actually you need to give at least the path from the addons

`/base/tests/test_configmanager.py`

but it is True that before adding the leading slash this tag could have matche a file

 `/x-base/tests/test_configmanager.py`

Reply:
Just an assertion test to fix and it should be fine.